code/c++/search_letor.cc

```diff
@@ -8,26 +29,12 @@ using namespace std;
 
 static void show_usage()
 {
-    cout << "Usage: rank_letor --db=DIRECTORY MODEL_METADATA_KEY QUERY\n";
+    cout << "Usage: search_letor --db=DBPATH MODEL_METADATA_KEY QUERY\n";
```

Comment:
`--db=` isn't expected here.

Reply:
I think db_path is expected here.

Reply:
Indeed, this wasn't addressed correctly - I've commented on the fixup commit.